camille/process/delta_temp.py

```diff
@@ -0,0 +1,5 @@
+
+def process(amb, sea):
+    delta_temp  = sea.resample('20T').mean() - amb.resample('20T').mean()
+    delta_temp.columns = ['delta_temp']
```

Comment:
This shouldn't be needed? Is this a renaming?

Reply:
Do we not want a column name for the returned Series?